Tests/AlicerceTests/Shared/CancelableBagTestCase.swift

```diff
@@ -0,0 +1,117 @@
+import XCTest
+@testable import Alicerce
+
+class CancelableBagTestCase: XCTestCase {
+
+    private var cancelable: CancelableBag!
+
+    override func setUp() {
+        super.setUp()
+
+        cancelable = CancelableBag()
+    }
+
+    override func tearDown() {
+        cancelable = nil
+
+        super.tearDown()
+    }
+
+    // init
+
+    func testInit_WithMultipleCancelables_ShouldSucceed() {
+        cancelable = CancelableBag([MockCancelable(), MockCancelable()])
+    }
+
+    // isCancelled
+
+    func testIsCancelled_WithNotCancelledCancelableBag_ShouldReturnFalse() {
+        XCTAssertFalse(cancelable.isCancelled)
+    }
+
+    func testIsCancelled_WithCancelledCancelableBag_ShouldReturnTrue() {
+        XCTAssertFalse(cancelable.isCancelled)
+        cancelable.cancel()
+        XCTAssertTrue(cancelable.isCancelled)
+    }
+
+    // add
+
+    func testAdd_WithNotCancelledCancelledBag_ShouldAddCancelableToBag() {
```

Comment:
Maybe use `WithNotCancelledCancelableBag` or simply `WithNotCancelledBag` as the `Bag`s in the test refer to `CancelableBag` anyway.

Reply:
I think I've messed up those names, yes 😆